datahub-graphql-core/src/main/resources/entity.graphql

```diff
@@ -1451,7 +1455,7 @@ type Role implements Entity {
     """
     Role properties to include Request Access Url
     """
-    properties: RoleProperties!
+    properties: RoleProperties
 
     """
     A standard Entity Type
```

Comment:
i'm getting an error when trying locally if I don't set any role actors on a role saying that this is a required field that was not set.

I'd make this field not required for graphql since we aren't always setting actors on the result as you can see in `RoleMapper.java`
```
        final EnvelopedAspect envelopedUsers = aspects.get(Constants.ROLE_ACTORS_ASPECT_NAME);
        if (envelopedUsers != null) {
            result.setActors(mapActor(new Actors(envelopedUsers.getValue().data())));
        }
```

Or alternatively we could always set an empty list of actors and always return it saying we don't have any actors. either way we need to prevent this error from popping up:
<img width="1475" alt="image" src="https://github.com/datahub-project/datahub/assets/28656603/d0d38ef4-400d-4cf2-938b-8f23ccea7cb0">

Reply:
updated